config.yaml

```diff
@@ -47,6 +48,7 @@ profiles:
       - kafka-unbundler-consumer
       - fhir-ig-importer
       - reprocess-mediator
+      - openfn
```

Comment:
I don't think this should be deployed by default for CDR, let's leave it as optional.